docs/2.build/1.chain-abstraction/what-is.md

```diff
@@ -1,61 +1,96 @@
 
-<hr subclass="subsection" />
+Central to this vision is the concept of **chain abstraction**, which is designed to provide seamless access to a wider range of crypto assets, smart contracts, and blockchain states. This is achieved through a trio of key technologies: the intent layer, [chain signatures](chain-signatures/chain-signatures.md), and the [omnibridge](#).
+
+![Chain Abstraction Stack](/docs/assets/chain-abstract-1.png)
+ 
+- The intent layer serves as a foundational framework for user requests.
+- [Chain signatures](chain-signatures/chain-signatures.md) empower smart contracts to sign transactions across different assets and chains.
+- [Omnibridge](#) facilitates communication and state transitions between various blockchains, ensuring that transactions are executed smoothly.
+
+---
+
+## Multi-chain signatures: One account, multiple chains
 
-## Multi-chain signatures: One account, multiple chains 
 Currently, users and applications are siloed in different chains. This means that a user needs to create a new account for each chain they want to use. This is not only cumbersome for the user, but also for the developer who needs to maintain different codebases for each chain.
 
 NEAR Protocol provides a multi-chain signature service that allows users to use their NEAR Account to sign transactions in **other chains**. This means that a user can use the same account to interact with **Ethereum**, **Binance Smart Chain**, **Avalanche**, and **NEAR**.
 
+![Chain Signatures](/docs/assets/chain-abstract-2.png)
+
 :::info
 Multi-chain signatures work by combining **smart contracts** that produce signatures, with indexers that listen for these signatures, and relayers that submit the transactions to other networks. This allows users to hold assets and use applications in **any** network, only needing to have a single NEAR account.
 :::
 
-<hr subclass="subsection" />
+---
 
-## Fast-Auth: Email onboarding
-One of the first barriers that new users face when entering the world of Web3 is the need to create a crypto wallet. This generally implies the need to choose a wallet, create and store a recovery phrase, and obtain deposit funds to start using the account.
+## Applications
 
-With FastAuth, users only need to provide an email address to create a NEAR account. Using the same email address the user will be able to use their account across applications and devices.
+Thanks to significant advancements in the cryptocurrency space, including cross-chain interactions, account management, and the raise of AI applications, truly innovative projects are being developed in the NEAR ecosystem:
 
-:::warning
-FastAuth is being deprecated, stay tuned for updates
+### Bitcoin Use Cases
```

Comment:
I wonder if we have links for all of these use-cases, I'm afraid right now it sounds super "marketing", and not oriented to helping devs learn something

Otherwise maybe we make a blog post?

Reply:
Interesting, Proximity has actually made a blogpost that I think talks about all of this https://medium.com/@ProximityFi/building-bitcoin-native-dapps-with-nears-abstraction-stack-fb4aeb448eee

Reply:
Good call, @bucanero I think you can draw some inspiration from this BP on better summarizing the CA stack here.